src/modules/client/components/clientAlerts/ClientAlert.tsx

```diff
@@ -0,0 +1,90 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
+import theme from '@/config/theme';
+import {
+  clientNameAllParts,
+  parseAndFormatDate,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import {
+  ClientAlertFieldsFragment,
+  ClientWithAlertFieldsFragment,
+} from '@/types/gqlTypes';
+
+export interface ClientAlertProps {
+  alert: ClientAlertFieldsFragment;
+  client: ClientWithAlertFieldsFragment;
```

Comment:
This confused me a little. This component only needs the `client` in order to show the client name. It doesn't _need_ to have the alerts hanging off the client. Changing the type to `client: ClientNameFragment` would be clearer. Typescript should be happy with it because the `ClientWithAlertFieldsFragment` includes the name fragment.

Reply:
I figured maybe it's less confusing to have the ClientAlert card just receive a string.

The `ClientWithAlertFieldsFragment` is still used in a couple of other places, though - should I get rid of those and replace them all with just `Client`?

Reply:
It makes sense to use `ClientWithAlertFieldsFragment` only when you need the `client` to have alerts on it, if that makes sense. Yeah you could also just accept a `clientName` string, I think thats also a good option.